sopel/bot.py

```diff
@@ -263,16 +295,22 @@ def register(self, callables, jobs, shutdowns, urls):
             self.register_url_callback(func.url_regex, func)
 
     def part(self, channel, msg=None):
-        """Part a channel."""
+        """Leave a channel.
+
+        :param str channel: the channel to leave
+        :param str msg: the message to display when leaving a channel
+        """
         self.write(['PART', channel], msg)
 
     def join(self, channel, password=None):
         """Join a channel
```

Comment:
```suggestion
        """Join a channel.
```

Didn't notice the missing `.` before. :/